src/openrct2-ui/windows/LoadSave.cpp

```diff
@@ -927,7 +927,7 @@ namespace OpenRCT2::Ui::Windows
 
         bool isSave = (type & 0x01) == LOADSAVETYPE_SAVE;
 
-        if ((type & 0x0E) == LOADSAVETYPE_HEIGHTMAP && !isSave)
```

Comment:
This bug made its way into v0.4.20, so it should be in the changelog, I’d say.

Reply:
Indeed, though I surprisingly couldn't get it to trigger in the release?? I'll need to re-check.